otter/integration/tests/test_lbsh.py

```diff
@@ -0,0 +1,100 @@
+"""
+Tests covering the Load Balancer self healing behaviors
+"""
+
+from __future__ import print_function
+
+from testtools.matchers import (
+    MatchesAll
+)
+
+from twisted.internet.defer import gatherResults, inlineCallbacks
+
+from twisted.trial import unittest
+
+from otter.integration.lib.cloud_load_balancer import (
+    ContainsAllIPs,
+    HasLength
+)
+from otter.integration.lib.resources import TestResources
+from otter.integration.lib.trial_tools import (
+    TestHelper,
+    get_identity,
+    get_resource_mapping,
+    region,
+    tag
+)
+
+timeout_default = 600
+
+
+class TestLoadBalancerSelfHealing(unittest.TestCase):
+    """
+    This class contains test cases to test the load balancer healing
+    functino of the Otter Converger.
+    """
+
+    def setUp(self):
+        """
+        Establish an HTTP conncetion pool and commonly used resources for each
```

Comment:
While this is true, the helper mostly does that, so maybe just "Establish resources used for each test, such as the auth token and a load balancer"?